csharp/src/Ice/Util.cs

```diff
@@ -106,6 +107,11 @@ public object Clone()
         /// The list of TypeId namespaces. Default is Ice.TypeId.
         /// </summary>
         public string[] typeIdNamespaces = { "Ice.TypeId" };
+
+        /// <summary>
+        /// The <see cref="SslClientAuthenticationOptions"/> used by the client-side ssl transport.
+        /// </summary>
+        public SslClientAuthenticationOptions sslClientAuthenticationOptions;
```

Comment:
@bernardnormier this is the InitializationData update.

Reply:
Ok, I missed it.

I would name this field clientAuthenticationOptions.